labelbox/schema/model_run.py

```diff
@@ -446,6 +449,66 @@ def export_labels(
                          self.uid)
             time.sleep(sleep_time)
 
+    """
+    Creates a model run export task with the given filter and returns the task.
+    
+    >>>    export_task = export_labels_v2("my_export_task", filter={"media_attributes": True})
+    
+    """
+
+    def export_labels_v2(self, task_name: str,
+                         filter: Optional[ModelRunExportFilter]) -> Task:
+        mutation_name = "exportDataRows"
+        create_task_query_str = """mutation exportDataRowsPyApi($input: ExportDataRowsInput!){
+          %s(input: $input) {taskId} }
+          """ % (mutation_name)
+        params = {
+            "input": {
+                "taskName": task_name,
+                "filters": {
+                    "modelRunIds": [self.uid],
+                    "projectIds": []
+                },
+                "params": {
+                    "includeAttachments":
+                        filter["attachments"]
+                        if filter and "attachments" in filter else False,
+                    "includeMediaAttributes":
+                        filter['media_attributes']
+                        if filter and 'media_attributes' in filter else False,
+                    "includeMetadata":
+                        filter['metadata_fields']
+                        if filter and 'metadata_fields' in filter else False,
```

Comment:
Maybe this would read better if at the top you did something like:
```
_filter = filter or {}
```
Then instead of all of these conditions you can just do:
```
"includeAttachments" : _filter.get('attachments', False)
```

Reply:
I like that idea